src/gui/window_header.c

```diff
@@ -58,7 +56,7 @@ void window_header_init(window_header_t *window) {
     window->icons[HEADER_ICON_WIFI] = HEADER_ISTATE_OFF;
     window->label = NULL;
 
-    if (media_is_inserted()) {
+    if (marlin_vars()->media_inserted) {
```

Comment:
what shall be displayed in case of a mount-error?

Reply:
now it is not solved, this is part of header/footer rework.